src/Runner.Common/ConfigurationStore.cs

```diff
@@ -78,8 +78,10 @@ public interface IConfigurationStore : IRunnerService
         bool IsServiceConfigured();
         bool HasCredentials();
         CredentialData GetCredentials();
+        CredentialData GetV2Credentials();
```

Comment:
we keep the existing `.credentials` file on disk, migration will create `.v2Credentials` file, if anything goes wrong with the new authorization setting in `v2Credentials`, we will fallback to `.credentials`